submissions/krailoartur/HTML_CSS_practice/style.css

```diff
@@ -0,0 +1,186 @@
+*:focus {
+    outline-offset: 3px;
+    outline: 1px solid #74b9ff;
+}
+
+body {
+    color: #636e72;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+nav ul {
```

Comment:
Avoid style by HTML tag or `#id`. Separate responsibilities:
- HTML tag for markup
- CSS class for styling
- ID for JS logic